CGPs/cgp-0071.md

```diff
@@ -0,0 +1,132 @@
+---
+cgp: 71
+title: Addings USDC oracles
+date-created: 2023-02-16
+author: Roman Croessmann(@rcroessmann)
+status: DRAFT
+discussions-to: [Multi-Collateral Mento forum post](https://forum.celo.org/t/mento-core-contract-upgrade-1-multi-collateral-mint/4942/4)
```

Comment:
This can also be a normal link instead of a MD style link as the square brackets create a Yaml error when displayed as a MD file.